src/data/agile-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const agileQuiz = [
+  {
+    Question: "What are the three primary roles in Scrum and their key responsibilities?",
+    Answer: "The three primary roles in Scrum are Product Owner, Scrum Master, and Team Member. The Product Owner maximizes ROI, the Scrum Master guides team performance, and Team Members have authority over work methods.",
```

Comment:
answer is to long.
You need to add answers that are short so they fit nicely in the answers container for the quiz.
You can find the live quiz site here to see what it looks like 
https://developerquiz.org/

```suggestion
    Answer: "Product Owner, Scrum Master, and Team Member",
```